### ISSUE TITLE
Error in dialogs afters upgrading to SDK 4.5.1

### ISSUE THREAD

**Issue by szul** [BotBuilderCommunity/botbuilder-community-js#62]
After the upgrade, the following errors are encountered when running `npm run test`:

```
C:\Users\micha\x\botbuilder-community-js\libraries\botbuilder-dialog-prompts\node_modules\botbuilder-dialogs\lib\prompts\oauthPrompt.js:235
            catch {
                  ^

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:607:28)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Module.require (module.js:587:17)
```

This seems to be pointing to a file in the botbuilder-dialog package in the Bot Framework.

**Comment by szul** [comment]
The upgraded 4.5 package was not published to NPM. On hold until this issue is resolved.

**Comment by szul** [comment]
This might be solved by 4.5.2.
